system.json

```diff
@@ -1,9 +1,9 @@
 {
     "auth": true,
-    "description": "",
-    "messaging_url": "platform.clearblade.com:1883",
-    "name": "PackageDev_jStat",
+    "description": "Statistical Analysis Toolkit Library for ClearBlade Platform",
+    "messaging_url": "staging.clearblade.com:1883",
+    "name": "dev-jstat-statistical-analysis",
```

Comment:
I recommend changing this value to 'jstat-statistical-analysis'

Reply:
60c64ee9a362a0d40a1dbe84bd0ed466450ae98c